index.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+#container { display: flex; }
+
+nav {
+  padding-top: 10px;
+}
+
+@media only screen and (max-width: 600px) {
+  nav {
+    display: flex;
+    justify-content: space-between;
+    top: 0;
+    width: 100%;
+    position: fixed;
+  }
+
+  .fa-bars {
+    color: blue;
+    padding-right: 20px;
+  }
+
+  .fa-bars:hover {
+    opacity: 0.5;
+  }
+
+  .catch-text-sec {
+    background-image: url(/assets/headerTitle.svg);
+    background-repeat: no-repeat;
+    height: 100vh;
+    background-size: cover;
+  }
+
+  #catch-text-heading {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 50px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    padding-top: 21%;
+    width: 327px;
+    height: 150px;
+  }
+
+  .about {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    font-style: normal;
+  }
+
+  .socials-header {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .socials-list {
+    display: block;
+    list-style: none;
+    margin-left: -50px;
+  }
+
+  .social-icon {
+    float: left;
+    margin-left: 10px;
+  }
```

Comment:
- [ ] Kindly dd space between the icons and resize them to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/204935010-c76753cc-4760-49c7-b995-61e154244b3a.png) | ![image](https://user-images.githubusercontent.com/58642949/204935064-2202c52f-b4e6-45f5-bab1-dfd28beac57d.png) |